src/conmon.c

```diff
@@ -280,7 +280,11 @@ int main(int argc, char *argv[])
 	 * Glib does not support SIGCHLD so use SIGUSR1 with the same semantic.  We will
 	 * catch SIGCHLD and raise(SIGUSR1) in the signal handler.
 	 */
-	g_unix_signal_add(SIGUSR1, on_sigusr1_cb, pid_to_handler);
+	struct pid_check_data data = {
```

Comment:
can we use a more informative variable name?

Reply:
I'll use `exit_status_cache` as @mrunalp suggested